readme.md

```diff
@@ -10,14 +10,25 @@ Written by Taylor Otwell as a part of Laravel's [Illuminate/Support](https://git
 
 Lovingly split by Matt Stauffer for [Tighten Co.](https://tighten.co/), with a kick in the butt to finally do it from [@assertchris](https://github.com/assertchris).
 
-## Installation
+## Install
 
 With [Composer](https://getcomposer.org):
 
 ```bash
 composer require tightenco/collect
 ```
 
+## Upgrade
```

Comment:
Please mention it defaults to the latest version of 5.7 but you can also pass a specific version

Reply:
Good call, I've just added it to the readme.